django_ses/__init__.py

```diff
@@ -201,6 +201,11 @@ def send_messages(self, email_messages):
                         message.extra_headers['request_id']
                     ))
 
+                signals.message_sent.send(
+                    sender=self.send_messages,
```

Comment:
The [Django docs](https://docs.djangoproject.com/en/5.1/topics/signals/#connecting-to-specific-signals) seem to suggest using a class, although maybe this would work. I wonder if the sender should be `SESBackend` though

Reply:
You're absolutely correct! It should be the class, else it won't be possible to filter correctly. I'll fix it!